### ISSUE TITLE
Have you all been to book thing?

### ISSUE THREAD

**Issue by bjornmeansbear** [GD399-OSD/library-identity#6]
http://bookthing.org/

Maybe something is surfaced by collecting books as materials for design experiments?

**Comment by johnvetter** [comment]
Woah I thought they had a fire! Are they open again?

**Comment by bjornmeansbear** [comment]
@johnvetter Yes, they were been fire damaged and closed, but they've been open again since September or October
